src/toxcore/tcp/codec.rs

```diff
@@ -154,8 +166,8 @@ impl Encoder for Codec {
         let mut encrypted_packet_buf = [0; MAX_TCP_ENC_PACKET_SIZE];
         let (_, encrypted_packet_size) = encrypted_packet.to_bytes((&mut encrypted_packet_buf, 0))
             .expect("EncryptedPacket serialize failed"); // there is nothing to fail since
-                    // serialized Packet is not longer than 2032 bytes
-                    // and we provided 2050 bytes for EncryptedPacket
+        // serialized Packet is not longer than 2032 bytes
```

Comment:
This comment must be connected to line `.expect("EncryptedPacket serialize failed"); // there is nothing to fail since`